web/src/app/admin/standard-answer/StandardAnswerCreationForm.tsx

```diff
@@ -86,12 +90,28 @@ export const StandardAnswerCreationForm = ({
         >
           {({ isSubmitting, values, setFieldValue }) => (
             <Form>
-              <TextFormField
-                name="keyword"
-                label="Keywords"
-                tooltip="If all specified keywords are in the question, then we will respond with the answer below"
-                placeholder="e.g. Wifi Password"
-                autoCompleteDisabled={true}
+              {values.matchRegex ? (
+                <TextFormField
+                  name="keyword"
+                  label="Regex pattern"
+                  isCode
+                  tooltip="Triggers if the question contains this regex pattern (matches Python `re.search()`)"
+                  placeholder="danswer/.*?/admin"
+                />
+              ) : (
+                <TextFormField
+                  name="keyword"
+                  label="Keywords or pattern"
+                  tooltip="Triggers if all specified keywords are in the question or if regex pattern exists in the question"
+                  placeholder="wifi"
```

Comment:
1. Does the placeholder "wifi" here make sense? Is there a more descriptive example we could use?
2. How should the user enter multiple keywords?
3. What's the difference between "keywords or pattern" and "regex pattern"?

Reply:
Oh, this still has the copy from the original PR. Let me reword these.

This branch should be for exact keyword matches only.

Reply:
Fixed, take a look.

Reply:
To answer your other questions:

- the user provides keywords as a string, separated by spaces. If desired (probably in a separate PR?), we could replace this with some kind of multi-input tag system (like https://eui.elastic.co/#/forms/combo-box) ?
- open to suggestions on what might be a good replacement for `wifi`. maybe something that reflects the usefulness on matching all keywords? like `google groups`?

Starting to exceed the scope of the PR but we could also add a toggle for "any keywords" vs "all keywords"

Reply:
Ya I was thinking as well that any vs all keyword toggle would be great as well. Generally we try not to scope creep but this seems fairly minor and since it was an obvious one to you as well, let's do it.

How do you feel about the use case "IT ticket" and something like "If you need any help from the IT team, please email internalsupport@company.com"

Reply:
`IT Ticket` and the suggested markdown placeholder sounds good. How do you feel about scoping this PR specifically to the regex flag and then figuring out the priority of the any/all keywords feature in a separate PR?

Reply:
Above suggestion sounds good.

Reply:
Talked with @yuhongsun96 and we'll work on that as a separate PR.